contrib/lib/src/uuid.rs

```diff
@@ -23,7 +23,7 @@ use std::ops::Deref;
 use rocket::request::{FromParam, FromFormValue};
 use rocket::http::RawStr;
 
-pub use self::uuid_crate::parser::ParseError;
+pub use self::uuid_crate::Error as ParseError;
```

Comment:
So, I think we could actually backwards compatibly support both `0.7` and `0.8` after all:

`type ParseError = <self::uuid_crate::Uuid as FromStr>::Err;`

(although `cargo` would still always choose `0.8` during a `cargo update` unless `--precise` is used)

Reply:
(cc @SergioBenitez)